packages/app/cypress/e2e/sidebar_navigation.cy.ts

```diff
@@ -235,6 +235,20 @@ describe('Sidebar Navigation', () => {
       cy.get('.router-link-active').findByText('Specs').should('be.visible')
     })
 
+    it('Specs sidebar nav link is not active when a test is running', () => {
+      cy.location('hash').should('equal', '#/specs')
+      cy.contains('.router-link-exact-active', 'Specs')
+
+      cy.findAllByTestId('spec-item').first().click()
+      cy.location('hash').should('contain', '#/specs/runner')
+      cy.contains('.router-link-exact-active', 'Specs').should('not.exist')
+      cy.percySnapshot()
+
+      cy.get('a[href="#/specs"]').click()
```

Comment:
does it make sense to use a more specific selector like `cy.findAllByTestId('nav-link-spec')` just in case another link was added later?

Reply:
The update looks elegant!